packages/arb-token-bridge-ui/src/util/wagmi/setup.ts

```diff
@@ -124,7 +125,15 @@ export function getProps(targetChainKey: string | null) {
     //
     // https://github.com/wagmi-dev/references/blob/main/packages/connectors/src/walletConnect.ts#L114
     getChains(sanitizeTargetChainKey(targetChainKey)),
-    [customInfuraProvider(), publicProvider()]
+    [
+      customInfuraProvider(),
+      publicProvider(),
+      jsonRpcProvider({
+        rpc: chain => ({
+          http: rpcURLs[chain.id]!
+        })
+      })
```

Comment:
i put a `try catch` in this `getProps` function and it was throwing an error of invalid provider for Ethereum Local

somehow it was never an issue until `wagmiAdditonalNetworks.ts` are imported by the other files simultaneously on the metadata PR